orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/MonitorWebhookTask.groovy

```diff
@@ -53,48 +53,54 @@ class MonitorWebhookTask implements OverridableTimeoutRetryableTask {
   @Autowired
   WebhookService webhookService
 
-  static requiredParameters = ["statusEndpoint", "statusJsonPath"]
-
   @Override
   TaskResult execute(Stage stage) {
-    def missing = requiredParameters.findAll { !stage.context.get(it) }
-    if (!missing.empty) {
-      throw new IllegalStateException("Missing required parameter${missing.size() > 1 ? 's' : ''} '${missing.join('\', \'')}'")
-    }
+    StageData stageData = stage.mapTo(StageData)
```

Comment:
did some minor refactor here to bring into ~2018... :)
`stageData` and no more throws from here -> return TERMINAL instead